packages/sewing-kit-koa/src/manifests.ts

```diff
@@ -0,0 +1,150 @@
+import {join} from 'path';
+
+import {readFile, readJson, pathExists} from 'fs-extra';
+import {matchesUA} from 'browserslist-useragent';
+import appRoot from 'app-root-path';
+import {ungzip} from 'node-gzip';
+
+import {ConsolidatedManifest, Manifest} from './types';
+
+const DEFAULT_MANIFEST_PATH = 'build/client/assets.json';
+const MULTI_ASYNC_LANGUAGE_MANIFESTS = 'noLocales';
+
+export interface ResolveOptions {
+  locale?: string | undefined;
+}
+
+export class Manifests {
+  manifestPath: string;
+  private resolvedManifestEntry?: Manifest;
+
+  constructor(manifestPath: string | null = null) {
+    this.manifestPath = manifestPath || DEFAULT_MANIFEST_PATH;
+  }
+
+  async getResolvedManifest(
```

Comment:
nit: `Manifests#getResolvedManifest` seems a bit redundant, we could shorten to something like `Manifests#get` or `Manifests#resolve`

Reply:
👍 Good call.  I went with resolve.